### PR TITLE
GH-1199 Adds Toolbar and CommandWidget for ACC

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes GH-1199

@dtsnell4 Please Review

http://rawgit.com/jbrundage/Visualization/AccurintCommandCenter/demos/dermatology.html?src/composite/CommandWidget

### CODE REVIEW

File: src/common/HTMLWidget.js [hunk 1]
```diff
@@ -106,7 +106,8 @@
     HTMLWidget.prototype.target = function (_) {
         if (!arguments.length) return this._target;
         if (this._target && _) {
-            throw "Target can only be assigned once.";
+            this._target = null;
```
Comment (mlzummo) [inline src/common/HTMLWidget.js]: yea i think this is better functionality

**Comment by jbrundage** [comment]
@GordonSmith @mlzummo The Toolbar widget changes should be isolated from some of the other changes in this PR. I'm going to create a separate issue and PR for the Toolbar.

**Comment by GordonSmith** [comment]
@jbrundage Can you rebase this PR now the toolbar changes have been pulled and squash the commits?

**Comment by mlzummo** [comment]
@jbrundage bump

**Comment by jbrundage** [comment]
@GordonSmith @mlzummo This PR contains an old version of the Toolbar widget. Closing this in favor of refactoring the "CommandWidget" and creating a new PR.

**Pull request closed without merging**